task/export.go

```diff
@@ -61,24 +63,75 @@ func TaskExport(tctx *TaskContext) (*TaskHandlerOutput, error) {
 	defer cancel()
 	tctx = tctx.WithContext(ctx)
 	content := tctx.Progress.TrackReader(file.Body, size, 1)
-	output, err := uploadFile(tctx, asset, content)
+
+	name := "asset-" + asset.PlaybackID
+	contentType := "video/" + asset.VideoSpec.Format
+	cid, internalMetadata, err := uploadFile(tctx, customParams, ipfsParams, name, content, contentType)
+	if err != nil {
+		return nil, err
+	}
+	nftCid, err := uploadNftMetadata(tctx, ipfsParams, cid)
+	if err != nil {
+		return nil, err
+	}
+	return &TaskHandlerOutput{
+		TaskOutput: &data.TaskOutput{
+			Export: &data.ExportTaskOutput{
+				Internal: internalMetadata,
+				IPFS: &data.IPFSExportInfo{
+					VideoFileCID:   cid,
+					NFTMetadataCID: nftCid,
+				},
+			},
+		},
+	}, nil
+}
+
+func TaskExportData(tctx *TaskContext) (*TaskHandlerOutput, error) {
```

Comment:
Would you mind moving this to a separate `export_data` file? Since we're not sharing the same entrypoint, mostly the upload file helpers, I think it couldbe simpler to keep them in separate files.

Reply:
No problem. Moved in 74b5325077d29a5bce09019e549e50453c201373